dataprofiler/tests/profilers/test_int_options.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import IntOptions
+from dataprofiler.tests.profilers.test_numerical_options import TestNumericalOptions
+
+class TestIntOptions(TestNumericalOptions):
+    
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.option_class = IntOptions
+    
+    def test_init(self, *mocks):
+        super().test_init(*mocks)
+    
+    def test_set_helper(self, *mocks):
+        super().test_set_helper(*mocks)
+
+    def test_set(self, *mocks):
+        super().test_set(*mocks)
+    
+    def test_validate_helper(self, *mocks):
+        super().test_validate_helper(*mocks)
+    
+    def test_validate(self, *mocks):
+        super().test_validate(*mocks)
+    
+    def test_is_numeric_stats_enabled(self, *mocks):
+        super().test_is_numeric_stats_enabled(*mocks)
```

Comment:
Similar to approach in BaseColumnOptions, taking the approach of overriding the super to call the super to explicitly state which tests are being run (for readability and future modification).